scripts/us_census/pep/us_pep_sexrace/postprocess.py

```diff
@@ -0,0 +1,374 @@
+        mcfflag (int) : flag value helps in generating output files.
+        Possible values are 1,2,3 and 4.
+        1 - generate mcf for files natioanl before the year 2000.
+        2 - generate mcf for files state and county after the year 2000.
+        3 - generate mcf for files state and county after the year 2000.
+        4 - generate mcf for files natioanl after the year 2000.
+
+    Returns:
+        None
+    """
+    mcf_template = """Node: dcid:{}
+typeOf: dcs:StatisticalVariable
+populationType: dcs:Person
+{}{}
+statType: dcs:measuredValue
+measuredProperty: dcs:count
+"""
+
+    final_mcf_template = ""
+    for sv in statvar_names:
+        if "Total" in sv:
+            continue
+        if "Year" in sv:
+            continue
+        if "geo_ID" in sv:
+            continue
+        gender = ''
+        race = ''
+        sv_prop = sv.split("_")
+        for prop in sv_prop:
+            if prop in ["Count", "Person"]:
+                continue
+            if "Male" in prop or "Female" in prop:
+                gender = "gender: dcs:" + prop
+            else:
+                race = "race: dcs:" + prop + "\n"
+        final_mcf_template += mcf_template.format(sv, race, gender) + "\n"
+    # Writing Genereated MCF to local path.
+    if mcfflag == Outputfiles.NationalBefore2000.value:
+        write_to_mcf("national_before_2000", final_mcf_template)
+    elif mcfflag == Outputfiles.StateCountyBefore2000.value:
+        write_to_mcf("state_county_before_2000", final_mcf_template)
+    elif mcfflag == Outputfiles.StateCountyAfter2000.value:
+        write_to_mcf("state_county_after_2000", final_mcf_template)
+    else:
+        write_to_mcf("national_after_2000", final_mcf_template)
+
+
+def generate_tmcf(df_cols: list, tmcfflag: int) -> None:
+    """
+    Function generate 4 TMCF
+    1. national_before_2000.tmcf : for the natioanl files before the year 2000.
+    2. state_county_before_2000.tmcf : for the state and county files before
+    the year 2000.
+    3. state_county_after_2000.tmcf : for the state and county files after
+    the year 2000.
+    4. national_after_2000.tmcf : for the natioanl files after the year 2000.
+
+    Args:
+        df_cols (list) : List of DataFrame Columns
+        tmcfflag (int) : flag value helps in generating output files.
+        Possible values are 1,2,3 and 4.
+        1 - generate tmcf for files natioanl before the year 2000.
+        2 - generate tmcf for files state and county after the year 2000.
+        3 - generate tmcf for files state and county after the year 2000.
+        4 - generate tmcf for files natioanl after the year 2000.
+
+    Returns:
+        None
+    """
+
+    tmcf_template = """Node: E:postprocess->E{}
+typeOf: dcs:StatVarObservation
+variableMeasured: dcs:{}
+measurementMethod: dcs:{}
+observationAbout: C:postprocess->geo_ID
+observationDate: C:postprocess->Year
+observationPeriod: "P1Y"
+value: C:postprocess->{} 
+
+"""
+    j = 0
+    measure = ""
```

Comment:
nit: usually we abbreviate 'measurement_method' to 'mmethod', instead of 'measure'.

Reply:
Converted 'measure' to 'mmethod'.